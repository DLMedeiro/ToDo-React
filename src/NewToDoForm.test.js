import React from "react";
import { render } from "@testing-library/react";
import NewToDoForm from "./NewToDoForm";

test("it renders without crashing", () => {
  render(<NewToDoForm />);
});

it("matches snapshot", function () {
  const { asFragment } = render(<NewToDoForm />);
  expect(asFragment()).toMatchSnapshot();
});
