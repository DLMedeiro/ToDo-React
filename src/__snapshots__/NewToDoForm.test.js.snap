// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <form>
    <label
      for="id"
    >
      id
    </label>
    <input
      id="id"
      name="id"
      placeholder="id"
      type="text"
      value=""
    />
    <label
      for="todo"
    >
      todo
    </label>
    <input
      id="todo"
      name="todo"
      placeholder="todo"
      type="text"
      value=""
    />
    <p />
    <p />
    <button>
      Add Todo Item
    </button>
  </form>
</DocumentFragment>
`;
